src/store/reducers/__tests__/modal.spec.js

```diff
@@ -0,0 +1,36 @@
+import store from 'store/store';
+import reducer from '../modal';
+import { openModal, closeModal } from 'store/actions';
+import { modalType } from 'components/Modal/Modal';
+
+describe('Modal reducer', () => {
+  const movieId = '1';
+
+  it('returns the initial state', () => {
+    expect(reducer(undefined, {})).toMatchSnapshot();
+  });
+
+  it('returns correct state when opening modal', () => {
+    const expectedState = {
+      currentModal: modalType.edit,
+      isModalShown: true,
+      movieId
+    };
+
+    expect(
+      reducer(undefined, openModal(modalType.edit, movieId))
```

Comment:
to properly test reducer you should pass result of an action and not an action itself. thus you will guarantee that with correct payload you will get expected result